mousestyles/data/__init__.py

```diff
@@ -1,15 +1,46 @@
 import os as _os
-
 import numpy as np
-
+import pandas as pd
 from mousestyles import data_dir
 
-def all_feature_data():
+def load_all_features():
+    """
+    Returns a (21131, 13) size pandas.DataFrame object corresponding to
+    9 features over each mouse's 2-hour time bin. The first four columns
+    index each mouses's 2-hour bin:
+
+    Column 0: the strain of the mouse (0-15)
+    Column 1: the mouse number (number depends on strain)
+    Column 2: the day number (5-16)
+    Column 3: 
 
-    feat_arr = ['ASProbability', 'ASNumbers', 'ASDurations', 'Food', 'Water', 'Distance', 'ASFoodIntensity', 'ASWaterIntensity', 'MoveASIntensity']
-    feat_arr_units = ['Active state probability', 'Number AS onsets', 'Total time [sec]', 'Food consumed [g]', 'Water consumed [mg]', 'Distance travelled [m]', 'ASFoodIntensity', 'ASWaterIntensity', 'MoveASIntensity [cm/ASsec]']
+    The remaining 9 columns are the computed features.
+
+    Returns
+    -------
+    features_data_frame : pandas.DataFrame
+        A dataframe of computed features.
+    """
+    features = ['ASProbability', 'ASNumbers', 'ASDurations', 'Food', 'Water', 'Distance', 'ASFoodIntensity', 'ASWaterIntensity', 'MoveASIntensity']
+
+    all_features = np.load(_os.path.join(data_dir, 'all_features_mousedays_11bins.npy'))  # 9 x 1921 x (3 labels + 11 feature time bins)
+   
+    # Here we begin reshaping the 3-d numpy array into a pandas 2-d dataframe
+    columns = ['strain', 'mouse', 'day']
+    for i in range(11):
```

Comment:
This whole loop could be replaced with `columns += list(range(11))` (as you know, I love avoiding loops at all costs :stuck_out_tongue_winking_eye:)